client/additional-methods-setup/wizard/README.md

```diff
@@ -0,0 +1,94 @@
+# Wizard helpers
```

Comment:
`/wizard` is the same as https://github.com/Automattic/woocommerce-payments/tree/develop/client/additional-methods-setup/wizard. The differences are:
- renamed stylesheet to `style.scss`
- renamed any occurrence of `woocommerce-payments` to `woocommerce-gateway-stripe`